ietf/templates/doc/document_material.html

```diff
@@ -126,7 +126,7 @@
         <div class="card-body">
             {% if doc.rev and content != None %}
                 {% if content_is_html %}
-                    {{ content|sanitize|safe }}
+                    {{ content }}
```

Comment:
I think this is not going to have the effect you intended. If I follow you correctly, you're changing this to count on the content string _not_ being marked safe, and would get the default treatment for an unsafe string. That default will escape all the html tags. So (for instance), https://datatracker.ietf.org/meeting/80/materials/agenda-80-dispatch would not render legibly?
If I'm missing the path that would cause the right thing to happen with such input, let me know.